client/image_save.go

```diff
@@ -1,18 +1,27 @@
 package client
 
 import (
+	"encoding/json"
 	"io"
 	"net/url"
 
+	"github.com/docker/docker/api/types"
+
 	"golang.org/x/net/context"
 )
 
 // ImageSave retrieves one or more images from the docker host as an io.ReadCloser.
 // It's up to the caller to store the images and close the stream.
-func (cli *Client) ImageSave(ctx context.Context, imageIDs []string) (io.ReadCloser, error) {
+func (cli *Client) ImageSave(ctx context.Context, images []string, opts types.ImageSaveOptions) (io.ReadCloser, error) {
 	query := url.Values{
-		"names": imageIDs,
+		"names": images,
+	}
+	query.Set("format", opts.Format)
+	refsJSON, err := json.Marshal(opts.Refs)
+	if err != nil {
+		return nil, err
 	}
+	query.Set("refs", string(refsJSON))
```

Comment:
Same here for `format` and `refs` options for `GET /images/get`) to the API; these should be documented in the [swagger.yml](https://github.com/moby/moby/blob/master/api/swagger.yaml), and in the [api version history](https://github.com/moby/moby/blob/master/docs/api/version-history.md)

Be sure to document what accepted values are for `format` (and if there's a default / what's the default)
  
Same here for documenting the `refs` option as well; what's the format? what's the purpose?

Reply:
will do

Reply:
Thanks!

Reply:
no swagger for experimental